package.json

```
@@ -29,7 +29,7 @@
     "copy-secrets": "cp secret.config.json.example secret.config.json",
     "test": "NODE_OPTIONS=--max-old-space-size=6144 jest",
     "silent-test": "NODE_OPTIONS=--max-old-space-size=8192 jest --silent",
-    "test:watch": "NODE_OPTIONS=--max-old-space-size=6144 jest --watchAll",
+    "test:watch": "NODE_OPTIONS=--max-old-space-size=6144 jest --watchAll --coverage=false",
```

```suggestion
    "test:watch": "NODE_OPTIONS=--max-old-space-size=6144 jest --watchAll",
```
Probably good to keep as is but, if you are trying to do it without I personally use a `alias edsc_test="npm test -- --watch --coverage=false"` in my zshell

---

Sorry, I forgot I had this in here. Removed ✅